app/views/auth_key_pair_cloud/_form.html.haml

```diff
@@ -37,11 +37,15 @@
         %label.col-md-2.control-label
           = _('Provider')
         .col-md-8
-          %select{:name                         => "ems_id",
+          = select_tag("ems_id",
+                  "",
                   "ng-model"                    => "keyPairModel.ems",
-                  "ng-options"                  => "ems.name for ems in ems_choices track by ems.id",
+                  "ng-options"                  => "ems.name for ems in ems_choices",
                   "id"                          => "ems_id",
-                  "required"                    => ""}
+                  "required"                    => "",
+                  "pf-select"                   => true)
+          :javascript
+            miqInitSelectPicker();
```

Comment:
@dgutride You can remove this JS method and instead use the directive `selectpicker-for-select-tag` for the `select_tag`